Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -324,7 +324,7 @@ public extension ChatMessage {
             return nil
         }
         
-        guard type == .regular || type == .reply else {
+        guard type == .regular || type == .reply || type == .deleted else {
             // Delivery status only makes sense for regular messages and thread replies.
```

Comment:
Let's change this comment :)

also would it make sense to actually provide helper var on top of the type?
`type.canShowDeliveryStatus` to something similar

Reply:
✅ Done